### PR TITLE
chore: Removed invalid links

### PR DESCRIPTION
**Pull request by yilimin999**
# What ❔

- Removed invalid links

## Why ❔

- The current link is inaccessible.

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [x] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

File: docs/repositories.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 | Internal repository                                         | Public repository                                                     | Description                                               |
```
Comment (popzxc) [inline docs/repositories.md:7]: Could you please remove the "Internal repository" column completely?

**Review by popzxc — COMMENTED** [review]
Thanks! One small request.

**Review by popzxc — APPROVED** [review]
Thanks!

**Comment by yilimin999** [comment]
Hello, can it be merged?

**Pull request merged**